ckanext/data_qld_theme/templates/user/new.html

```diff
@@ -0,0 +1,8 @@
+{% ckan_extends %}
+
+  {% block help_inner %}
+  <h2 class="module-heading">{{ _('Why Sign Up?') }}</h2>
+  <div class="module-content">
+    <p>{% trans %}Users must register to make data requests, comment against data requests and to give dataset feedback.{% endtrans %}</p>
```

Comment:
Inconsistent use of "to"; the second list term omits it but the third includes it.

Reply:
This was copied from AC 1. b. i. in the story https://salsadigital.atlassian.net/browse/DQL-50